package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "website",
+  "version": "1.0.0",
```

Comment:
semantic versioning: "1.0.0" means this is an official release. Use another version.

Reply:
Changed to 1.0.0-alpha.1 for marking as pre-release, according to semver:
- https://softwareengineering.stackexchange.com/questions/275358/version-number-for-a-pre-release-software-that-will-be-the-second-major-release
- https://semver.npmjs.com/